src/apps/cost_matrix_based_matching/online_localizer_lsh.cpp

```diff
@@ -70,12 +69,11 @@ int main(int argc, char *argv[]) {
   parser.parseYaml(config_file);
   parser.print();
 
-  std::unique_ptr<OnlineDatabase> database =
-      std::make_unique<CostMatrixDatabase>(
-          /*costMatrixFile=*/parser.costMatrix,
-          /*queryFeaturesDir=*/parser.path2qu,
-          /*refFeaturesDir=*/parser.path2ref, /*type=*/FeatureType::Cnn_Feature,
-          /*bufferSize=*/parser.bufferSize);
+  std::unique_ptr<OnlineDatabase> database = std::make_unique<OnlineDatabase>(
+      /*queryFeaturesDir=*/parser.path2qu,
+      /*refFeaturesDir=*/parser.path2ref, /*type=*/FeatureType::Cnn_Feature,
```

Comment:
nit: as always, I'm not a fan of shortening names. I would prefer `path_to_query` and `path_to_reference` instead or even more precise names. Then you might be able to drop the comments 🤷

Reply:
It is also in the config.yaml file. I will keep this in mind and change when restructuring the yaml logic.
Thanks!